modules/common/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/metrics/StatsDReporter.scala

```diff
@@ -88,7 +89,11 @@ object StatsDReporter {
         FailedInsertCountName -> lms.failedInsertCount.toString,
         BadCountName          -> lms.badCount.toString
       ) ++ lms.latency.map(l => LatencyGaugeName -> l.toString)
-    case rms: RepeaterMetricsSnapshot => List(UninsertableName -> rms.uninsertableCount.toString)
+    case rms: RepeaterMetricsSnapshot =>
+      List(
+        UninsertableName -> rms.uninsertableCount.toString,
+        RepeaterInsertedName -> rms.insertedCount.toString
```

Comment:
@istreeter Count of the events which inserted by repeater wasn't sent to Statsd previously. I change it. Do you it is okay ?